config/ember-try.js

```diff
@@ -145,20 +116,7 @@ module.exports = async function () {
           devDependencies: {},
         },
       },
-      embroider.safe,
-      // disable embroider optimized test scenarios, as the dynamism these
-      // tests use is not compatible with embroider we are still exploring
-      // appropriate paths forward.
-      //
-      // Steps to re-enable:
-      //
-      // 1. have a strategy to make this work
-      // 2. uncomment the next line
-      // embroider.optimized,
-      //
-      // 3. add "embroider-optimized" to .github/workflows/ci-build.yml's
-      //    ember-try-scenario list.
-      //
+      embroiderSafe(),
```

Comment:
how's embroiderOptimized() ?

Reply:
That is not working because of https://github.com/embroider-build/embroider/issues/522, as of now!

Reply:
Ah, wait, actually I managed to get it to work. Kind of missed one of the comments in between :sweat_smile: